voluptuous/schema_builder.py

```diff
@@ -1017,16 +1017,75 @@ def wrapper(*args, **kwargs):
 
     return decorator
 
+def _args_to_dict(func, args):
+    """
+        Returns argument names as values as key-value pairs.
+
+        :input `func` that is a function object
```

Comment:
If you want to document the parameters it should be in correct [RST](http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html) format.